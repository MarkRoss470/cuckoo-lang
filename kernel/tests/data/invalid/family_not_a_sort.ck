data SomeType : Type where

data Invalid : SomeType where