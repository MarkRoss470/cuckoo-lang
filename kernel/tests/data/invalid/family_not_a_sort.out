ERROR at 3:16-3:24: SomeType is not a sort or family of sorts.